packages/terra-button/src/_mixins.scss

```diff
@@ -27,6 +27,11 @@
     color: map-get($color-map, active-color);
   }
 
+  &:disabled,
+  &.is-disabled {
+    background-color: map-get($color-map, disabled-background-color);
```

Comment:
Missing the original issue of providing themability to the text **color**, mainly in utility style button, not exclusively background-colors. After reviewing recently updated other theme designs, this may no longer be needed. Confirming before merging.

Reply:
Confirmed that in addition to background-color, for full control of disabled styling, we should add to the color map:
```
disabled-border-color
disabled-box-shadow
disabled-color
```
For default-theme, match the disabled values to the same current normal values.

Reply:
Above color maps have been added.

Reply:
Marking Resolved.